client/components/swipeable/index.js

```diff
@@ -95,6 +95,9 @@ export const Swipeable = ( {
 
 	useLayoutEffect( () => {
 		if ( ! updateEnabled ) {
+			if ( pagesStyle?.height ) {
+				setPagesStyle( { ...pagesStyle, height: false } );
```

Comment:
1. (request change) We need to link to this PR as a comment because it seems hard to understand this at first glance without the context.

2. My personal preference for clearing out this key is using `undefined` as that's what gets returned if there really isn't a `height` key.
```suggestion
				setPagesStyle( { ...pagesStyle, height: undefined } );
```

Reply:
Great suggestions, @candy02058912, I will make both.